From 0783909b19bb4b393d2893a8107e4bac36611fd3 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Fri, 24 Aug 2018 19:19:10 -0700
Subject: [PATCH] display: Define soong namespace

HALs are starting to move to blueprint files, which means that pathmap
ifeqs no longer work. This results in modules with the same name in
different platform HAL directories. Since these modules aren't used or
linked anywhere outside the HAL directories and a device's device.mk,
we can safely use a soong namespace to isolate the different HAL
directories' blueprints.

Change-Id: Ibe275bd3183af68c36a516d24f3792ec520e60bc
---
 Android.bp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Android.bp b/Android.bp
index 5b2f3a9bd..14e37af39 100644
--- a/Android.bp
+++ b/Android.bp
@@ -1,3 +1,6 @@
+soong_namespace {
+}
+
 cc_defaults {
     name: "display_defaults",
     cflags: [
-- 
2.17.1

